testing/scenario_app/lib/main.dart

```diff
@@ -43,6 +43,8 @@ Map<String, Scenario> _scenarios = <String, Scenario>{
 };
```

Comment:
some of the golden images look like it didn't work, like the opacity and rotate ones

Reply:
these might be actual bugs in the implementation of virtual display. The screenshots are correct in the sense that running the scenario manually produces the same output. I'd investigate further, but it's outside of the scope of this PR.  I=